### DISCUSSION TITLE
Does S3 versioning work with longhorn volume backup?

### DISCUSSION THREAD

**Discussion by mike12806** [longhorn/longhorn#9461] [Q&A]
I'm seeing 2 "empty" backup volumes being presented in the longhorn UI, and I'm wondering if this has to do with S3 versioning being enabled on my local minio setup (to support bucket replication). Previously I was using AWS S3 without bucket versioning and never saw this. 

I checked the docs and don't see it listed as a requirement or unsupported feature in the backup documentation - but curious if anyone has seen a similiar issue. 

Wanted to check here first before opening an issue. 

Thanks,

Mike

**Comment by shuo-wu** [comment]
Close this since there is a ticket https://github.com/longhorn/longhorn/issues/9462 tracking this
